qutebrowser/app.py

```diff
@@ -479,11 +479,9 @@ def _init_modules(*, args):
 
     with debug.log_time("init", "Initializing SQL/history"):
         try:
-            log.init.debug("Initializing SQL...")
-            sql.init(os.path.join(standarddir.data(), 'history.sqlite'))
-
             log.init.debug("Initializing web history...")
-            history.init(objects.qapp)
+            history.init(db_path=os.path.join(standarddir.data(), 'history.sqlite'),
```

Comment:
```suggestion
            history.init(db_path=pathlib.Path(standarddir.data()) / 'history.sqlite',
```

Will make the migration easier once we migrate `standarddir` to pathlib (#176).

Then `sql.init` and potentially `sql.Database` would need to take a `pathlib.Path`, but I'd prefer already doing this now that we're refactoring things anyways.

Reply:
Ah, I was messing with this yesterday (calling `str(path)` on every test before passing it to `Database.__init__`). I guess I can make the whole change now since I'm here.

Reply:
Done. I left `sql.Database.__init__` as it is because it also needs to accept `:memory:`.